src/main/java/mate/academy/spring/model/dto/request/OrderRequestDto.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.spring.model.dto.request;
+
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class OrderRequestDto {
```

Comment:
Where you use this dto?

Reply:
Nowhere I just forgot to delete this, sorry.